### ISSUE TITLE
Mounted name went wrong when there's a space on Linux and Darwin

### ISSUE THREAD

**Issue by kimlimjustin** [cristiammercado/node-disk-info#17]
Mounted name of a disk when wrong when there's a space, in my case, `New Volume` become `New`. I've read the code and figured out this bug occurred because the code directly passed the `tokens[5]` which is an array came from splitting space in the `line` variable into the `drive_1` class, and I found a way to fix it!  I'd like to be assigned to fix this bug if you like to :)

Regards

**Comment by Ekristoffe** [comment]
Indeed, does this do the trick:
Replace tokens[5] by this :
`const mount = tokens.slice(5).join(' ');`

**Comment by kimlimjustin** [comment]
> Indeed, does this do the trick:
> Replace tokens[5] by this :
> `const mount = tokens.slice(5).join(' ');`

Yeah, I figured it out and applied into my project, may I open an PR for this issue here?

**Comment by Ekristoffe** [comment]
yeah why not but I have a PR which may bother. anyway no need to ask you can just do it :)

**Comment by kimlimjustin** [comment]
> yeah why not but I have a PR which may bother. anyway no need to ask you can just do it :)

Okay thanks!

**Comment by Ekristoffe** [comment]
#18 was made in the dist files (compiled files) which mean each time someone will compile from source your fix would be lost.
I added them in the src folder and made #20
